drivers/abcc.js

```diff
@@ -1,26 +1,39 @@
+const Driver = require('../models/driver');
 const request = require('../lib/request');
 const Ticker = require('../models/ticker');
 const { parseToFloat } = require('../lib/utils.js');
 
-module.exports = async () => {
-  const tickers = await request('https://api.abcc.com/api/v2/tickers.json');
-  const pairs = Object.keys(tickers);
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class Abcc extends Driver {
+  /**
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
```

Comment:
I think the description here is redundant, we might want to look into keeping this as simple as possible. Perhaps even the return tag could just be defined in the driver class.

Reply:
We will look at this later when we start automatically generating documentation from these comments.